tests/framework/CodeInterfaceTests/CobraTF/test1_cobratf_code_interface.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" ?>
+<Simulation>
+  <TestInfo>
+    <name>framework/CodeInterfaceTests/CobraTF.test1</name>
+    <author>jyoo</author>
+    <created>2018-04-03</created>
+    <classesTested>Models.Code.CobraTF</classesTested>
+    <description>
+       An example of using the CobraTF code interface. This test is aimed to test the mechanics of the interface using a MonteCarlo sampling strategy.
+    </description>
+    <revisions>
+      <revision author="jyoo" date="2018-04-03">New CobraTF test case</revision>
+    </revisions>
+  </TestInfo>
+  <RunInfo>
+    <WorkingDir>CobraTFinterfaceTest1</WorkingDir>
+    <Sequence>testDummyStep_MC</Sequence>
+    <batchSize>1</batchSize>
+  </RunInfo>
+
+  <Files>
+    <Input name="cobra_input" type="" >case5-15.inp</Input>
+  </Files>
+
+  <Models>
+    <Code name="MyCobraTF" subType="CobraTF">
+      <executable>
+        <!--  /path/to/relap/executable/relap5.x
```

Comment:
this test is not using relap5

Reply:
It's fixed.

Reply:
It looks to me that is not...please fix this